nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -686,7 +685,7 @@ void nxagentChangeKeyboardControl(DeviceIntPtr pDev, KeybdCtrl *ctrl)
 
     value_mask = KBLed | KBLedMode;
 
-    for (i = 1; i <= 32; i++)
+    for (int i = 1; i <= 32; i++)
```

Comment:
Uhm there definitely is one, or how else would `i = 1` work? I've seen it at the start of the scope.